tests/test_rwc_jazz.py

```diff
@@ -96,10 +121,20 @@ def test_load_metadata():
         'track_number': 'Tr. 04',
         'title': 'Crescent Serenade (Piano Solo)',
         'artist': 'Makoto Nakamura',
-        'duration_sec': '02:47',
+        'duration': 167,
         'variation': 'Instrumentation 1',
         'instruments': 'Pf',
     }
+    assert metadata['RM-J044'] == {
```

Comment:
Another one of the tests is saying `KeyError: 'RM-J044'` - maybe this key needs to be added to the partial metadata file in the tests?